### PR TITLE
Upgrade vue-cli-plugin-electron-builder and Release

### PR DESCRIPTION
**Pull request by baruchiro**
Related to https://github.com/brafdlog/budget-tracking/discussions/189 and https://github.com/nklayman/vue-cli-plugin-electron-builder/issues/1229

### CODE REVIEW

File: test/e2e/specs/launch.spec.ts
Comment (baruchiro) [inline test/e2e/specs/launch.spec.ts:72]: There are a lot of types of problems with browser interactions.

Maybe it because I changed the Jest Env from JSDOM to Node, I don't know, it is out of scope for me. We will take care of the tests later (I hope!)

File: vue.config.js
Comment (baruchiro) [inline vue.config.js:20]: Just note, this is what allows us to call `fs`, for example, from the renderer process.

We talked about it once, we may one day want to invest in canceling it.

File: README.md
Comment (baruchiro) [inline README.md:96]: Read this

> ## Release
> 
> The Release workflow, configured in `.github/workflows/release.yml`, will draft a new release or update the artifacts on an existing draft, according to the following conditions:
> 
> 1. A change pushed to the `master` branch.
> 1. The `version` from the `package.json`, with the prefix `v`, is not a **published** release.
> 1. Upload the artifacts to the `v${package_version}` Github Release.
>

File: README.md
Comment (brafdlog) [inline README.md:1]: Nope. The original capitalization is correct

File: test/e2e/specs/launch.spec.ts
Comment (brafdlog) [inline test/e2e/specs/launch.spec.ts:11]: Why do we need this again? Worth writing a comment

**Review by brafdlog — APPROVED** [review]
Nice. See comments

File: README.md
Comment (baruchiro) [reply README.md:1]: Not arguing. I thought in the title it can be different, but I don't trust myself.

Comment (baruchiro) [reply README.md:1]: Not arguing. I thought in the title it can be different, but I don't trust myself.

Comment (baruchiro) [inline README.md:1]:
```suggestion
# Automated expense tracking from Israeli banks and credit cards
```

File: test/e2e/specs/launch.spec.ts
Comment (baruchiro) [reply test/e2e/specs/launch.spec.ts:11]: I don't know why we need to increase, it something to investigate.
But I very want to keep the test, even if it just tests the "title", because it tests the e2e.

**Comment by baruchiro** [comment]
@brafdlog Please merge if no other comments.

After merging, wait for [release](https://github.com/brafdlog/budget-tracking/actions/workflows/release.yml) flow to finish, and then go to the [Releases](https://github.com/brafdlog/budget-tracking/releases) and publish the existing draft.

After that, you may upgrade the version in the `package.json`. (You don't have to do that, I will do that myself next week)

**Pull request merged**
